README.md

```diff
@@ -41,6 +41,16 @@ Source-build solves common challenges that most developers encounter when trying
 
 Starting with .NET 6, the core source-build infrastructure is integrated into the [dotnet/installer](https://github.com/dotnet/installer/tree/main/src/SourceBuild) repo. The `main` branch on this repo now contains the tooling needed to build .NET's external dependencies from source.
 
+## Linux distributions building and offering .NET
+| Distro |.NET Packages|Maintainer|Feed|Preview/Beta Packages|
+|---|---|---|---|---|
+| Alpine | [.NET in Alpine](https://pkgs.alpinelinux.org/packages?name=dotnet*&branch=v3.16&repo=&arch=&maintainer=)   | Linux Distro | Community  |  |
+| Arch Linux |  [.NET in Arch Linux](https://archlinux.org/packages/?q=dotnet)  | Linux Distro | Community  |   |
+| Arch Linux User Repository |  [.NET in Arch Linux User Repo](https://aur.archlinux.org/packages?K=dotnet)  | Community |Not Applicable|   |
```

Comment:
I wasn't expecting this. How is this different than Arch Linux?

Reply:
It's sort of similar to Ubuntu's universe (or main) vs PPAs. The Arch User Repository (aur) is fully contributed by users and hasn't gone through any sort of official testing or review.

https://wiki.archlinux.org/title/Arch_User_Repository:

> AUR packages are user-produced content. These PKGBUILDs are completely unofficial and have not been thoroughly vetted. Any use of the provided files is at your own risk.

Reply:
Right I get that.  My question is if this should be considered two different distros or two package feeds of the same distro?  To me `Arch Linux User Repository` isn't a distro.  Would it make sense to list the Arch_User_Repository in the Prevew/Beta Packages column (possibly renamed)?

Reply:
> Right I get that. My question is if this should be considered two different distros or two package feeds of the same distro? To me `Arch Linux User Repository` isn't a distro. Would it make sense to list the Arch_User_Repository in the Prevew/Beta Packages column (possibly renamed)?

Removing make more sense as those are unofficial packages being added by Arch Users (use at own risk sort of) as opposed to PPA which are beta packages ultimately making as the final supported version in the ubuntu.